packages/console/src/pages/Mfa/MfaForm/utils.ts

```diff
@@ -1,17 +1,25 @@
-import { MfaFactor } from '@logto/schemas';
+import { MfaFactor, MfaPolicy } from '@logto/schemas';
 import { conditional } from '@silverhand/essentials';
 
 import { type MfaConfig, type MfaConfigForm } from '../types';
 
 export const convertMfaConfigToForm = ({ policy, factors }: MfaConfig): MfaConfigForm => ({
-  policy,
+  isMandatory: policy === MfaPolicy.Mandatory,
+  setUpPrompt: [
+    MfaPolicy.NoPrompt,
+    MfaPolicy.PromptAtSignInAndSignUp,
+    MfaPolicy.PromptOnlyAtSignIn,
+  ].includes(policy)
```

Comment:
```ts
const signInPrompt = new Set<MfaPolicy>([
    MfaPolicy.NoPrompt,
    MfaPolicy.PromptAtSignInAndSignUp,
    MfaPolicy.PromptOnlyAtSignIn,
  ]):
```

P.S. If `policy === MfaPolicy.Mandatory` this field should be undefined?  Only fallback to `MfaPolicy.PromptAtSignInAndSignUp` if `policy === MfaPolicy.UserControled`

Reply:
Keep the default value to `PromptAtSignInAndSignUp` can make things easy when user "switch" the require option to false, then the "Select" will appear with a proper default value.

Reply:
Can you give an example? Those form data format util methods you have should handle these returned data type already.  Feels like this way is more accurate.

Reply:
So I can select both `Mandatory` and a prompt option in the `MfaPolicy` (no prompt, sign-in only, both), right?

Reply:
Please see line 32